README.md

```diff
@@ -43,8 +43,8 @@ Learn more at [http://openwhisk.apache.org](http://openwhisk.apache.org).
 
 The easiest way to start using OpenWhisk is to install the "Standalone" OpenWhisk stack.
 This is a full-featured OpenWhisk stack running as a Java process for convenience.
-Serverless functions run within Docker containers. You will need [Docker](https://docs.docker.com/install),
-[Java](https://java.com/en/download/help/download_options.xml) and [Node.js](https://nodejs.org) available on your machine.
+Serverless functions run within Docker containers. You will need [Docker](https://docs.docker.com/install), the
+[Java Development Kit](https://www.oracle.com/java/technologies/downloads/) and [Node.js](https://nodejs.org) available on your machine.
```

Comment:
Our policy is to reference OpenJDK as a basis and would also like to see a recommendation for a known, tested AdoptOpenJDK LTS version

Reply:
I do have openJDK 11.0.11.9, it does not work and I don't know where it came from.